uber/tests/models/test_badge_funcs.py

```diff
@@ -420,3 +427,21 @@ def test_out_of_badge_type(self, session, monkeypatch, before_printed_badge_dead
         session.regular_attendee.badge_type = c.STAFF_BADGE
         session.regular_attendee.badge_num = None
         assert 'There are no more badges available for that type' == check(session.regular_attendee)
+
+
+class TestDupeFixes:
+    @pytest.fixture(autouse=True)
+    def create_dupe_nums(self, session, monkeypatch):
+        session.staff_five.badge_num = 1
+        # Skip the badge adjustments here, which prevent us from setting duplicate numbers
+        monkeypatch.setattr(Attendee, '_badge_adjustments', 0)
+        session.commit()
+        session.staff_five.badge_num = None
```

Comment:
This simulates what would happen if an admin chose "Next badge num" on the admin form. Otherwise, we were testing the (impossible) case of an admin successfully setting a badge with a duplicate number to its own number - our model checks prevent this.